rskj-core/src/main/java/org/ethereum/vm/VM.java

```diff
@@ -1436,6 +1436,31 @@ protected void doJUMPDEST()
         program.step();
     }
 
+    protected void doMCOPY() {
+        if (computeGas) {
+            // See "Gas Cost" section on EIP 5656
+            // gas cost = 3 * (length + 31) + memory expansion cost + very low
+            long length = stack.get(stack.size() - 3).longValue();
+            long newMemSize = memNeeded(stack.peek(), length);
+            long cost = 3 * (length + 31) + calcMemGas(oldMemSize, newMemSize, 0) + 3; // TODO -> Check copy size
+
+            gasCost = GasCost.add(gasCost, cost);
+            spendOpCodeGas();
+        }
+
+        // EXECUTION PHASE
+        DataWord dst = program.stackPop();
+        DataWord src = program.stackPop();
+        DataWord length = program.stackPop();
+
+        if (isLogEnabled) {
+            hint = "dst: " + dst + " src: " + src + " length: " + length;
```

Comment:
suggestion:

I noticed that you are not logging at the end, maybe would be good to log these parameters that you set.

Reply:
Hey! yup, the thing is that `hint` is a global variable that will be logged by `logOpCode` automatically in case `isLogEnabled` is set to `true`. The function is called always inside the `steps` function just before updating the `vmCounter`.